src/lib/constants.js

```diff
@@ -1,4 +1,5 @@
 export const COLUMN_COUNT_QUERY_KEY = 'columns';
-export const LOCAL_STORAGE_NAMESPACE = 'music-markdown';
-export const REPO_REGEX = '/repos/:repo([^/]+/[^/]+)';
+export const WINDOW_STORAGE_NAMESPACE = 'music-markdown';
+export const REPOS_RESOURCE = 'repos';
```

Comment:
I think you can get rid of this now. It looks like noise in the code and I'm not sure it provides any value.